### DISCUSSION TITLE
ScriptReference/Random.Range

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#49] [Notes]
# ScriptReference/Random.Range

https://docs.unity3d.com/ScriptReference/Random.Range.html

**Comment by vertxxyz** [comment]
Due to `0` [signing](https://docs.unity3d.com/ScriptReference/Mathf.Sign.html) to `1`, and `Random.Range(int, int)` having an exclusive upper bound you can use:
`Mathf.Sign(Random.Range(-1, 1));` to generate a `-1` or `1`.
